This patch ensures that the Desktop Preferences .desktop file is not installed.

Author: Eugenio "g7" Paolantonio <me@medesimo.eu>
----
Index: pcmanfm/data/Makefile.am
===================================================================
--- pcmanfm.orig/data/Makefile.am
+++ pcmanfm/data/Makefile.am
@@ -7,7 +7,6 @@ man_MANS=pcmanfm.1
 desktopdir=$(datadir)/applications
 desktop_in_files= \
 	pcmanfm.desktop.in \
-	pcmanfm-desktop-pref.desktop.in \
 	$(NULL)
 desktop_DATA = $(desktop_in_files:.desktop.in=.desktop)
 @INTLTOOL_DESKTOP_RULE@
